src/robusta/cli/main.py

```diff
@@ -87,11 +87,16 @@ def set_pod_configs_for_small_clusters(self):
         )
 
 
-def guess_cluster_name():
+def guess_cluster_name(context):
     with click_spinner.spinner():
         try:
             all_contexts, current_context = config.list_kube_config_contexts()
+            if context is not None:
+                for i in range(len(all_contexts)):
+                    if all_contexts[i].get('name') == context:
+                        return context
```

Comment:
this returns the context name, not the cluster name (they can be different based on how the context is defined)
for example `kubectl config --kubeconfig=config-demo set-context dev-frontend --cluster=development --namespace=frontend --user=developer`
to return the actual cluster name you would want:
`all_contexts[i].get('context').get('cluster')` 
to return the cluster with that name

Reply:
currently for default case without ```--context``` flag we return the ```current_context.get('name')``` so are we suppose to change that as well into returning the name of cluster

Reply:
yea, it looks like a bug